diluvian/config.py

```diff
@@ -174,6 +174,16 @@ class NetworkConfig(BaseConfig):
         ones. If data is anisotropic and Z should only be downsampled every
         other level, this value could be [2, 1, 1]. Axes set to 0 are never
         downsampled.
+    unet_downsample_mode: string
+        The mode to use for downsampling. The two options are "fixed_rate",
+        which will use the downsample rate previously defined, and "as_needed",
```

Comment:
Would `"isotropy_approximating"` be a more clear and precise name for this mode? Would changing the mode name create many problems for you downstream if you decide to rebase/merge upstream diluvian after these PRs?

Reply:
I changed the name in the documentation. The implementation uses "isotropy_approximation" if anything other than the default "fixed_rate" option is specified. I'm not sure if this is the clearest way to do things but it means that this change doesn't really have any impact on upstream PRs